### ISSUE TITLE
Django 2.0 support

### ISSUE THREAD

**Issue by EugeneVC** [xaralis/django-static-sitemaps#58]
Hello.

In django v 2.0 i have error.

```
File "/home/eugene/VirtualEnv/Django2/lib/python3.4/site-packages/static_sitemaps/generator.py", line 18, in <module>
    from django.core.urlresolvers import reverse, NoReverseMatch
ImportError: No module named 'django.core.urlresolvers'
```

Please fix in file: static_sitemaps/generator.py

```
try:
    from django.core.urlresolvers import reverse, NoReverseMatch
except ImportError:
    from django.urls import reverse, NoReverseMatch
```

Thanks.

**Comment by BarnabasSzabolcs** [comment]
Hi xaralis,
I've checked the Django 2.1rc1 compatibility and the resolution to the bug is actually as simple as changing the import statements: https://stackoverflow.com/questions/43139081/importerror-no-module-named-django-core-urlresolvers

you only need to change 2 lines, so it should be 2 minutes time. Can you please fix this?

Thanks a lot,
Barney

**Comment by xaralis** [comment]
@BarnabasSzabolcs Should be fixed by a390779b9b6e4a44633875459a38658bb04ab37c, released as https://github.com/xaralis/django-static-sitemaps/releases/tag/4.5.0.

**Comment by BarnabasSzabolcs** [comment]
@xaralis awesome, thanks man!
